### PR TITLE
solution

### PR DESCRIPTION
**Pull request by VladBrynzia**
[DEMO LINK](https://vladbrynzia.github.io/react_movies-list-filter/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
 
 export const App: React.FC = () => {
+  const [findMovie, setFindMovie] = useState('');
+  const copyMovie = [...moviesFromServer];
+
+  const findYourMovie = useMemo(() => copyMovie.filter(movie => {
+    const findTitle = movie.title.toLowerCase().includes(findMovie.toLowerCase());
+    const findDescription = movie.description.toLowerCase().includes(findMovie.toLowerCase());
+    /* eslint-disable-next-line */
+    console.log(findMovie);
```
Comment (Olexandr-Trubin-Koshyk) [inline src/App.tsx:14]: Redundant `console.log`

**Review by Olexandr-Trubin-Koshyk — APPROVED** [review]
Good start!

File: src/App.tsx (hunk 1 above)
Comment (Ihor-Karpyn) [inline src/App.tsx:14]: :harold
```suggestion
```

File: src/App.tsx [hunk 2]
```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [findMovie, setFindMovie] = useState('');
+  const copyMovie = [...moviesFromServer];
```
Comment (Ihor-Karpyn) [inline src/App.tsx:8]: the filter isn't a mutable method
```suggestion
```

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**